src/platforms/unity/native-support/index.mdx

```diff
@@ -4,15 +4,21 @@ description: "Learn how the Unity SDK handles native support."
 sidebar_order: 100
 ---
 
+Currently, the Unity SDK offers native support for `Android`, `iOS`, and `Windows`. The support is `enabled` by default and you can opt out for each platform individually in the configuration window.
```

Comment:
Same comment as above. For "enabled", since we're talking about a configuration setting specifically, code format is okay here, but the wording just has to be tweaked a bit to support that.

```suggestion
Currently, the Unity SDK offers native support for Android, iOS, and Windows. The support is set to `enabled` by default and you can opt out for each platform individually in the configuration window.
```